assets/javascripts/kitten/components/simulators/loan-simulator.js

```diff
@@ -103,26 +110,61 @@ class LoanSimulator extends React.Component {
       return this.props.amountOutOfBoundsError
   }
 
-  installmentMin() {
-    const installmentStep = this.installmentStep()
-    const value = this.state.amount / this.props.durationMax
-    const min = Math.ceil(value / installmentStep) * installmentStep
-    if (min > this.state.amount * 1)
-      return this.state.amount
-    else
-      return min
+  render() {
+    return (
+      <LoanSimulatorContent
+        { ...this.props }
+        { ...this.state }
+        handleFocus={ this.handleFocus }
+        handleAmountChange={ this.handleAmountChange }
+        handleAmountKeyDown={ this.handleAmountKeyDown }
```

Comment:
Ça pue un peu ça. En fait il faudrait faire `onFoo={ this.handleFoo }` (et pas `handleFoo={ this.handleFoo }`).